### PR TITLE
Remove websocket code

### PR DESCRIPTION
**Pull request by Nutomic**
I was hoping to get rid of the `Perform` traits. But then API handlers would have to call `SendActivity` directly, which doesnt move unless we move all of it to the same crate. So for now Im simply removing the websocket code without any other changes. Maybe in the future we can improve this further.

### CODE REVIEW

File: crates/api_common/Cargo.toml [hunk 1]
```diff
@@ -14,20 +14,19 @@ path = "src/lib.rs"
...
+full = ["tracing", "rosetta-i18n", "chrono", "lemmy_utils",
     "lemmy_db_views/full", "lemmy_db_views_actor/full", "lemmy_db_views_moderator/full",
     "percent-encoding", "encoding", "reqwest-middleware", "webpage", "ts-rs"]
 
 [dependencies]
 lemmy_db_views = { workspace = true }
 lemmy_db_views_moderator = { workspace = true }
 lemmy_db_views_actor = { workspace = true }
-lemmy_db_schema = { workspace = true, default-features = false }
+lemmy_db_schema = { workspace = true }
```
[the remark this answers is not in the document]
Comment (dessalines) [reply crates/api_common/Cargo.toml:25]: Other than that it looks good.

[the remark this answers is not in the document]
Comment (Nutomic) [reply crates/api_common/Cargo.toml:25]: Yes this was throwing a compiler warning because the crate doesnt have any default features.

**Review by dessalines — APPROVED** [review]

**Comment by Nutomic** [comment]
CI seems to be building the wrong code again, its passing locally.

**Comment by dessalines** [comment]
I'll try restarting.

**Pull request merged**